mindsdb/api/mysql/mysql_proxy/classes/sql_query.py

```diff
@@ -65,6 +65,12 @@ def parse_insert(sql):
         values = p['literal'] if isinstance(p, dict) else p
         if isinstance(values, list) is False:
             values = [values]
+        for i, v in enumerate(values):
+            if isinstance(v, dict) and 'literal' in v:
+                v = v['literal']
+            elif v == 'null':
+                v = None
+            values[i] = v
```

Comment:
in theory `v` can be undeclared because the `else-if` statement above doesn't have `else`

Reply:
If `v` is not 'null' and not dict then it should not be changed. By default parser returns strings, so i think it correct here

Reply:
right, I didn't see the for loop